lib/Dialect/Calyx/Export/CalyxEmitter.cpp

```diff
@@ -1003,20 +1007,39 @@ void Emitter::emitLibraryPrimTypedByFirstOutputPort(
            << LParen() << bitWidth << RParen() << semicolonEndL();
 }
 
-void Emitter::emitLibraryFloatingPoint(Operation *op) {
-  auto cell = cast<CellInterface>(op);
-  // magic number for the index of `left/right` input port for
-  // `AddF`/`MulF`/`CompareF`; `in` input port for `FpToInt`.
+unsigned getFPBitWidth(Operation *op, CellInterface &cell) {
```

Comment:
Nit: can we just get the `CellInterface` cast inside the function rather than pass two parameters?